modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/resources/META-INF/resources/js/ApplicationsMenu.js

```diff
@@ -400,7 +412,7 @@ const ApplicationsMenu = ({
 				onMouseOver={fetchCategories}
 				small
 				symbol="grid"
-				title={Liferay.Language.get('applications-menu')}
+				title={OPEN_MENU_TITLE_TPL}
```

Comment:
I think we are still using the legacy tooltip here. I tried to replicate this on https://storybook.clayui.com/?path=/story/components-claytooltip--claytooltipprovider, but can't even get it to render with template literals. https://github.com/liferay/clay/issues/3652 won't help here.

Reply:
There should be no such thing as legacy tooltips. `.lfr-portal-tooltip` will trigger tooltips, but it uses `ClayTooltip`.

Reply:
Here: https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.js#L219

Reply:
Thanks @jbalsas, I thought React would strip whitespace for us. Does `dangerouslySetInnerHTML` disable that?

Reply:
That's likely, yeah... maybe we should strip it ourselves before passing it in there... 🤔

Reply:
This makes things a little more complicated 😐

Reply:
I don't think always stripping whitespace is a good idea. Right now if you use `title={'Line 1\nLine 2'}`, Line 1 and Line 2 will be on separate lines. What if we add an attribute like `data-tooltip-whitespace-normal` and output slightly different markup?
```
<div className="text-wrap">
  <div
    dangerouslySetInnerHTML={{
      __html:
        state.target.title ||
        state.target.dataset.restoreTitle ||
        state.target.dataset.title,
    }}
  />
</div>
```